datafile/openers/luarocks.lua

```diff
@@ -16,6 +16,26 @@ end
 local load_local_manifest = manif_core.load_local_manifest  -- LuaRocks 2
 load_local_manifest = load_local_manifest or manif_core.fast_load_local_manifest  --LuaRocks 3
 
+local cfg
+ok, cfg = pcall(require, "luarocks.cfg") -- LuaRocks 2
+if not ok then
+   _, cfg = pcall(require, "luarocks.core.cfg") -- LuaRocks 3
+end
```

Comment:
Shouldn't the checks be flipped?
```suggestion
ok, cfg = pcall(require, "luarocks.core.cfg") -- LuaRocks 3
if not ok then
   _, cfg = pcall(require, "luarocks.cfg") -- LuaRocks 2
end
```
LuaRocks 3 if more likely to have a stray LuaRocks 2 file around, than the other way around. Just to be safe.

Reply:
I missed the check above. Nvm.